src/index.css

```diff
@@ -11,3 +11,43 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+
+.p-button {
+  transition: transform 0.33s !important;
+}
+
+.p-button:hover {
+  transform: scale(1.02);
+}
+
+.p-avatar {
+  transition: transform 0.33s !important;
+}
+
+.p-avatar:hover {
+  transform: scale(1.2);
+}
+
+.align-center {
+  text-align: center;
+}
+
+.navbar a {
+  text-decoration: none;
+  color: skyblue;
+  font-size: 18px;;
+  padding-right: 20px;
+}
+
+.navbar {
+  margin-bottom: 100px;
+  background: white;
+}
+.w-50 {
+  max-width: 50vw;
+  margin: 10px auto;
+  text-align: center;
+  line-height: 30px;
```

Comment:
Can these be achieve using PrimeReact pre-made classes

Reply:
I did try with it wasnt able to perform. I thoght too push a basic skeleton and then get into css properties

Reply:
Did you look at the NavBar in PrimeReact? Looks like it has a component with this all done and then we can focus on adding features to it https://www.primefaces.org/primereact/showcase/#/menubar

<img width="1390" alt="Screenshot 2021-09-29 at 11 07 01" src="https://user-images.githubusercontent.com/624760/135248601-de1cfda6-3720-4630-a464-0d16665463d8.png">

Reply:
Okay..I will simply add this one ...then we can think which parts should be removed and which should be kept